.eslintrc.await.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
No, it can't be done as a magic comment:
https://github.com/eslint/eslint/discussions/14854?sort=top